### ISSUE TITLE
Can't open developer tools

### ISSUE THREAD

**Issue by antonioprates** [microsoft/vscode#129103]
Issue Type: <b>Bug</b>

I am using Code-OSS, not sure if this is to be reported here. Anyway, have installed it from official packages into Manjaro-Linux, as VSC is not available unless from AUR build. But I guess this could be related to Electron update. This issue happens even with all extensions disabled.

Version: 1.58.0
Commit: 2d23c42a936db1c7b3b06f918cde29561cc47cd6
Date: 2021-07-12T23:51:36.303Z
Electron: 12.0.14
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Linux x64 5.10.49-1-MANJARO

Steps to reproduce:
1. Help > Toggle Developer Tools (or shortcut CTRL+SHIFT+I)
2. Nothing happens... (or it asks to install formatter for current file!?, but then nothing happens)

VS Code version: Code - OSS 1.58.0 (2d23c42a936db1c7b3b06f918cde29561cc47cd6, 2021-07-12T23:51:36.303Z)
OS version: Linux x64 5.10.49-1-MANJARO
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-7267U CPU @ 3.10GHz (4 x 3500)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>rasterization: disabled_software<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|15.54GB (5.42GB free)|
|Process Argv||
|Screen Reader|yes|
|VM|0%|
|DESKTOP_SESSION|xfce|
|XDG_CURRENT_DESKTOP|XFCE|
|XDG_SESSION_DESKTOP|xfce|
|XDG_SESSION_TYPE|x11|
</details>Extensions: none

**Comment by IllusionMH** [comment]
/duplicate of #120380 see issues linked in comments for details.

TL;DR; If you need to open (Chrome) DevTools for running editor with shortcut - you need to remove condition.

**Comment by antonioprates** [comment]
It's not the same issue, I have read the other ones, even removing the when condition on the keybindings doesn't fix, it also does not open from the Help menu :/

**Comment by antonioprates** [comment]
@IllusionMH could you please reopen this issue, and pleas untag it as duplicate, as it's not...

**Comment by IllusionMH** [comment]
@antonioprates sorry about that, missed that it's not only about shortcut.

@deepak1556 could you please reopen this issue and do you know if it's know problem with DevTools/Electron?

**Comment by deepak1556** [comment]
I thought the issue was with vscode built from source but later realized you are referring to this package https://discover.manjaro.org/applications/code. It is better you report the error to the package maintainers as we don't officially maintain it.

FWIW I am unable to repro the issue with vscode built from source and on our official packages as well. If you launch with `code --verbose` it can give hint to where the problem lies.

Closing for now, can reopen if the issue is identified to be in core or the runtime.

**Comment by antonioprates** [comment]
Thanks @deepak1556 will report it to the package mantainers.
